packages/@lightningjs/ui-components-test-utils/package.json

```diff
@@ -8,11 +8,16 @@
   },
   "homepage": "https://rdkcentral.github.io/Lightning-UI-Components/",
   "type": "module",
-  "exports": "./index.js",
-  "files": [
-    "index.js",
-    "src/**"
-  ],
+  "exports": {
+    ".": {
+      "import": "./dist/lightning-ui-components-test-utils.min.mjs",
+      "require": "./dist/lightning-ui-components-test-utils.min.cjs",
+      "types": "./dist/lightning-ui-components-test-utils.d.ts"
+    },
+    "./src/": "./src/"
+  },
+  "main": "./dist/lightning-ui-components-test-utils.min.mjs",
+  "types": "./dist/lightning-ui-components-test-utils.d.ts",
```

Comment:
I don't see a `dist` folder generated in the test-utils dir. I think we need to also add a build script.

Reply:
Good call. I tried adding this to match ui-components, but after `yarn build` I still don't see a dist folder 🤔 
```json
  "scripts": {
    "build": "rm -rf ./dist && rollup --config ../../../rollup/rollup.config.js --name lightning-ui-components-test-utils"
  },